src/einsteinpy/examples.py

```diff
@@ -1,13 +1,14 @@
 import numpy as np
+from astropy import units as u
 
-from einsteinpy.coordinates.utils import four_position, stacked_vec
-from einsteinpy.geodesic import Geodesic
+from einsteinpy.coordinates import SphericalDifferential
+from einsteinpy.geodesic import TimelikeGeodesic
```

Comment:
Okay this will change , but not now

Reply:
I have changed the name to `Timelike`, as it's pretty evident, that we are talking about a time-like geodesic, when we write: `from einsteinpy.geodesic import Timelike`. As for Null Geodesics, we can either go with `Nulllike` or `NullGeodesic`, although the former is more "symmetrical" to `Timelike`, so to speak.